api/client/cli.go

```diff
@@ -125,6 +125,12 @@ func (cli *DockerCli) ImagesFormat() string {
 	return cli.configFile.ImagesFormat
 }
 
+// ServiceInspectFormat returns the format string specified in the configuration.
+// String contains columns and format specification, for example {{ID}}\t{{Name}}.
+func (cli *DockerCli) ServiceInspectFormat() string {
+	return cli.configFile.ServiceInspectFormat
+}
```

Comment:
I made a similar comment on another PR but I guess we haven't cleaned this up yet. I don't think we should be exposing the formatter directly from the `DockerCli` object.

We already have `cli.ConfigFile()` we should just use that and access the `ServiceInspectFormat` directly from the configFile object.